assets/js/admin/course-theme-sidebar.js

```diff
@@ -3,49 +3,152 @@
+					}
+				}
+			}
+		}
+
+		return {};
+	} );
+
+	let previewUrl = '';
+	if ( firstLesson?.id && currentPost?.id ) {
+		if (
+			globalLearningModeEnabled ||
+			currentPost.meta._course_theme === SENSEI_THEME
+		) {
+			previewUrl = `/?p=${ firstLesson.id }`;
+		} else {
+			previewUrl = `/?p=${ firstLesson.id }&learn=1&${ SENSEI_PREVIEW_QUERY }=${ currentPost.id }`;
+		}
+
+		if ( firstLesson.draft ) {
+			previewUrl = `${ previewUrl }&post_type=lesson`;
+		}
+	}
+
+	let customizerUrl = '';
+	if ( previewUrl ) {
+		customizerUrl = `/wp-admin/customize.php?url=${ encodeURIComponent(
+			previewUrl
+		) }`;
+	}
+
+	return { previewUrl, customizerUrl };
+};
 
 /**
  * Course Theme Sidebar component.
  */
 const CourseThemeSidebar = () => {
+	const globalLearningModeEnabled =
+		window.sensei?.senseiSettings?.sensei_learning_mode_all || false;
 	const [ theme, setTheme ] = useCourseMeta( '_course_theme' );
+	const { previewUrl, customizerUrl } = usePreviewAndCustomizerLinks();
 
 	return (
 		<PluginDocumentSettingPanel
 			name="sensei-course-theme"
-			title={ __( 'Course Theme', 'sensei-lms' ) }
+			title={ __( 'Course Styles', 'sensei-lms' ) }
 		>
-			<p>
-				{ __(
-					'This does not change the theme of your site. It only applies to logged in users when viewing the course.',
-					'sensei-lms'
-				) }
-			</p>
-			<SelectControl
-				label={ __( 'Theme', 'sensei-lms' ) }
-				hideLabelFromVision
-				value={ theme }
-				options={ [
-					{
-						label: __( 'Sensei Theme', 'sensei-lms' ),
-						value: SENSEI_THEME,
-					},
-					{
-						label: __( 'WordPress Theme', 'sensei-lms' ),
-						value: WORDPRESS_THEME,
-					},
-				] }
-				onChange={ setTheme }
-			/>
+			{ globalLearningModeEnabled ? (
+				<p className="sensei-course-theme-toggle__global-overwrite-notice">
+					<a href="/wp-admin/admin.php?page=sensei-settings#course-settings">
+						{ __(
+							'Learning Mode is enabled globally.',
+							'sensei-lms'
+						) }
+					</a>
+				</p>
+			) : (
+				<ToggleControl
+					label={ __( 'Learning mode', 'sensei-lms' ) }
+					help={
+						<>
+							<p className="sensei-course-theme-toggle__description">
+								{ __(
+									'Enable this mode to show an immersive and dedicated view for the course, lessons, and quizzes.',
+									'sensei-lms'
+								) }
+							</p>
+							{ previewUrl && (
+								<p className="sensei-course-theme-toggle__preview">
+									<a
+										href={ previewUrl }
+										className="sensei-course-theme-toggle__preview__link"
+										target="_blank"
+										rel="noopener noreferrer"
+									>
+										{ __( 'Preview', 'sensei-lms' ) }
+									</a>
+								</p>
+							) }
+						</>
+					}
+					checked={ theme === SENSEI_THEME }
+					onChange={ ( checked ) =>
+						setTheme( checked ? SENSEI_THEME : WORDPRESS_THEME )
+					}
+				/>
+			) }
+			{ customizerUrl && (
+				<p className="sensei-course-theme-toggle__customize">
+					<a
+						href={ customizerUrl }
```

Comment:
Similar to the preview, maybe we could also open the customizer in target blank by default. WDYT?

Reply:
The way I setup them is the way it works in WordPress by default. The regular post preview link in WordPress opens in a separate tab and the customize link opens in the same tab. So I replicated the same behavior.

I think the reason for this behavior in WordPress is that in the customizer mode the user can see the changes they are making directly. But in preview mode you need to switch to editor to make changes and then switch to preview to see the updates. It makes more sense to keep the preview and editor in two separate tabs. But there is no benefit in keeping the customizer in a separate tab, because you have both the changes and the outcome at the same place.

But let me know if you think it's better if we open the customizer in a separate window and I will update it. 👍